package.json

```diff
@@ -1,6 +1,8 @@
 {
   "name": "circles-contracts",
-  "version": "1.0.0",
+  "version": "1.0.2",
+  "author": "Sarah Friend (http://isthisa.com)",
+  "license": "AGPL",
   "preinstall": "nvm use -i 10.15.0 || echo nvm not found: check node version && pause",
```

Comment:
Is this still needed? If so, I would use a `.node-version` file instead and leave a comment in the README.md requirements. I use this project with Node Version 11, no problems here, fyi.

Reply:
I'm not/have never been using this, and I'm not sure who was ... probably fine to take out